framework/config/src/main/java/org/apache/cloudstack/framework/config/impl/ConfigDepotImpl.java

```diff
@@ -83,12 +87,17 @@ public class ConfigDepotImpl implements ConfigDepot, ConfigDepotAdmin {
     List<ScopedConfigStorage> _scopedStorages;
     Set<Configurable> _configured = Collections.synchronizedSet(new HashSet<Configurable>());
     Set<String> newConfigs = Collections.synchronizedSet(new HashSet<>());
+    Cache<String, String> configCache;
 
     private HashMap<String, Pair<String, ConfigKey<?>>> _allKeys = new HashMap<String, Pair<String, ConfigKey<?>>>(1007);
 
     HashMap<ConfigKey.Scope, Set<ConfigKey<?>>> _scopeLevelConfigsMap = new HashMap<ConfigKey.Scope, Set<ConfigKey<?>>>();
 
     public ConfigDepotImpl() {
+        configCache = Caffeine.newBuilder()
```

Comment:
Minor - in the next iteration, could we move/refactor caching library object creation & configuration to a separate class/utility/framework; so users only need to import & call the method to get the Cache objects/entities, but configuration etc are in that utility - making it easier to change the caching library in future. Otherwise LGTM, thanks @shwstppr